ballerina/README.md

```diff
@@ -1,17 +1,79 @@
-## Overview
+# Running Tests
```

Comment:
This should go under the `tests` directory. The `README` under the Ballerina directory should be the API documentation (Setup guide, Quick Start Guide, etc.)